themes/bootstrap3/templates/Auth/AbstractBase/login.phtml

```diff
@@ -19,7 +19,7 @@
         <a class="btn btn-link createAccountLink" href="<?=$this->url('myresearch-account') ?>?auth_method=<?=$account->getAuthMethod()?>"><?=$this->transEsc('Create New Account')?></a>
       <?php endif; ?>
       <?php if ($account->supportsRecovery()): ?>
-        <a class="btn btn-link" href="<?=$this->url('myresearch-recover') ?>?auth_method=<?=$account->getAuthMethod()?>"><?=$this->transEsc('Forgot Password')?></a>
+        <a class="btn btn-link recoverAccountLink" href="<?=$this->url('myresearch-recover') ?>?auth_method=<?=$account->getAuthMethod()?>"><?=$this->transEsc('Forgot Password')?></a>
```

Comment:
I'd use the dash-separated style for classes (`recover-account-link`).

Reply:
I did this for consistency with createAccountLink on the line above. I can change it, but then the question is whether I should change that one too. I think these are only really used for testing purposes, though I guess it's conceivable that somebody might use them for custom styling.

Reply:
In any case, I've changed it as you suggest for now, but I'm leaving existing things alone. :-)